### PR TITLE
Add interface check for Minikube

### PR DESCRIPTION
**Pull request by maelk**
Check if the interfaces exist before adding them instead of
checking if Minikube VM exists

### CODE REVIEW

File: 01_prepare_host.sh [hunk 1]
```diff
@@ -84,14 +84,19 @@ function init_minikube() {
...
-      # so just attach it and make it reboot. As long as the
-      # 02_configure_host.sh script does not run, the provisioning network does
-      # not exist. Attempting to start Minikube will fail until it is created.
+      sudo su -l -c "minikube stop" "$USER"
+    fi
+
+    # The interface doesn't appear in the minikube VM with --live,
+    # so just attach it and make it reboot. As long as the
+    # 02_configure_host.sh script does not run, the provisioning network does
+    # not exist. Attempting to start Minikube will fail until it is created.
```
Comment (russellb) [inline 01_prepare_host.sh]: I think this comment block should move down to before the `virsh attach-interface` command

Comment (maelk) [reply 01_prepare_host.sh]: fixed, thank you

**Pull request merged**